randovania/games/samus_returns/json_data/Area 6.txt

```diff
@@ -555,7 +555,22 @@ Extra - asset_id: collision_camera_043
   > Door to Diggernaut Shaft
       After Area 6 - Diggernaut and Lay Power Bomb
   > Before Diggernaut
-      Bomb and Phase Drift and Plasma Beam and Space Jump and Spring Ball and Shoot Beam Burst and Use Spider Ball
+      All of the following:
+          Lay Bomb
+          Any of the following:
+              # Energy Requirements
+              Combat (Hypermode) or Normal Damage ≥ 499
+              Combat (Beginner) and Normal Damage ≥ 399
+              Combat (Intermediate) and Normal Damage ≥ 299
+              Combat (Advanced) and Normal Damage ≥ 199
```

Comment:
health is adjusted for the 1.5 multiplier, right? asking since on my playthrough i had some troubles with 4 tanks, but with the 1.5 multiplier this would be 749 health  which is fine.

Reply:
Yes it is